audience_manager/form-tracker.rst

```diff
@@ -3,99 +3,135 @@
 
 Audience Manager Form Tracker API
 =================================
-Form Tracker allows to gather data from all forms on your site in one place.
+Form Tracker gathers data submitted via forms on your page and sends it to the Audience Manager :term:`user` profile as
+:term:`attributes<attribute>`.
+
+.. note::
+
+    Any :term:`attribute` updated this way will be categorized as :term:`Soft Data` and will be stored in profile
+    `attributes` (separate from profile `analytics`).
 
 Supported browsers
 ------------------
-All modern browsers (Chrome, Firefox, Safari, Edge). Internet Explorer from version 8 and above.
+All modern browsers: Chrome, Firefox, Safari, Edge. Internet Explorer from version 8 and above.
+
+.. todo:: Move information about supported browsers to document about PPMS in general since it's common to whole system.
 
 Private information
 -------------------
-Form tracker is trying to automatically detect fields containing private information and exclude them. Following fields
-are always removed from tracked data:
+Form Tracker is trying to automatically detect fields containing :term:`user` private information and ignores them.
+Following data is never send to Audience Manager:
+
+.. todo:: Check what is legal term for ignored information's.
 
-* "password" or "hidden" inputs
-* Credit card number
-* Credit card validation code
+- Value from input with ``password`` or ``hidden`` type.
+- Credit card number (heuristic detection).
+- Credit card validation code (heuristic detection).
+- Data from ignored fields (see :ref:`AM-optional-configuration`).
+
+.. note::
+
+    Heuristic detection makes best effort to automatically detect and ignore mentioned field types, but it doesn't
+    guarantee success. Additionally payment forms usually contain more fields for private information (e.g. address)
+    so it's recommended to ignore such forms using :ref:`AM-optional-configuration`.
 
 Loading snippet
```

Comment:
Lack of information about adding the snippet via Tag Manager. It should be mentioned here that we have it + maybe a link to the help center: https://help.piwik.pro/audience-manager/capturing-data-forms/